WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -31,19 +33,72 @@ import ZendeskSDK
         ZendeskUtils.sharedInstance.enableZendesk(true)
     }
 
-    static func isZendeskEnabled() -> Bool {
-        return UserDefaults.standard.bool(forKey: UserDefaultsKeys.zendeskEnabled)
+    static func createIdentity(with accountSettings: AccountSettings) {
+        let zendeskIdentity = ZDKAnonymousIdentity()
+
+        var userName = accountSettings.username
+        if accountSettings.firstName.count > 0 || accountSettings.lastName.count > 0 {
+            userName = accountSettings.firstName + " " + accountSettings.lastName
+        }
+
+        zendeskIdentity.email = accountSettings.email
+        zendeskIdentity.name = userName
+        ZDKConfig.instance().userIdentity = zendeskIdentity
+        ZendeskUtils.identityCreated = true
+    }
+
+    static func showHelpCenter(from navController: UINavigationController) {
+
+        if !ZendeskUtils.identityCreated {
+            return
+        }
+
+        let helpCenterContentModel = ZDKHelpCenterOverviewContentModel.defaultContent()
```

Comment:
Maybe a just a code style opinion, but would it make sense to unwrap `helpCenterContentModel` vs having an optional?  As far as I can see, it will never be `nil`, and passing `nil` to `ZDKHelpCenter.pushOverview` pretty much breaks ZenDesk.